### PR TITLE
[9.0][MIG][account_renumber] Migrate to 9.0, refactor.

### PR DESCRIPTION
**Pull request by yajo**
- In v9 `account.period` has disappeared, so now the wizard asks for start and end dates.
- `ir.sequence` has changed logic too, this module is now adapted.
- Old tests had to be run through the CLI. New tests work with standard test system.
- Old menu parent does not exist anymore, so it is now moved to other place.
- Old license & copyright headers are now updated, respecting old copyrights where found.
- `.pot` file removed.
- README added.

@Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
Yay! 100% :raised_hands: :tada:

File: account_renumber/wizard/wizard_renumber_view.xml [hunk 1]
```diff
@@ -1,51 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <openerp>
```
Comment (yvaucher) [inline account_renumber/wizard/wizard_renumber_view.xml]: as you removed `<data>` you probably want to rename this as `<odoo>` as well ?

File: account_renumber/wizard/wizard_renumber_view.xml [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
+                </group>
+                <group string="Journals to consider">
+                    <field name="journal_ids" editable="top"/>
+                </group>
+                <footer>
+                    <button name="renumber"
+                            string="Renumber"
+                            type="object"
+                            class="oe_highlight"/>
+                    or
```
Comment (yvaucher) [inline account_renumber/wizard/wizard_renumber_view.xml]: In 9.0 the `or` term was removed in all official wizards

**Comment by yvaucher** [comment]
Just 2 nitpickings but already :+1: (code review)

**Comment by rafaelbn** [comment]
Hi @Yajo we need to review coverage.

**Comment by antespi** [comment]
Got this error

```
Odoo
Odoo Server Error
Traceback (most recent call last):
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 647, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 684, in dispatch
    result = self._call_function(**self.params)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 320, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/service/model.py", line 118, in wrapper
    return f(dbname, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 313, in checked_call
    result = self.endpoint(*a, **kw)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 963, in __call__
    return self.method(*args, **kw)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/http.py", line 513, in response_wrap
    response = f(*args, **kw)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/addons/web/controllers/main.py", line 902, in call_button
    action = self._call_kw(model, method, args, {})
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/addons/web/controllers/main.py", line 890, in _call_kw
    return getattr(request.registry.get(model), method)(request.cr, request.uid, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/api.py", line 381, in old_api
    result = method(recs, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/addons/account_renumber/wizard/wizard_renumber.py", line 87, in renumber
    date_range.number_next = self.number_next
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/fields.py", line 842, in __set__
    record.ensure_one()
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3151820-379-69db2b/openerp/models.py", line 5415, in ensure_one
    raise ValueError("Expected singleton: %s" % self)
ValueError: Expected singleton: ir.sequence.date_range()
```

when doing:
- Got to Invoicing > Adviser > Renumber journal entries
- Select all journals
- Click 'Renumber'

No data nor entries changed or manipulated

**Comment by yajo** [comment]
It should be fixed now.

**Comment by carlosdauden** [comment]
:+1:

**Pull request merged**